seatunnel-config/src/test/java/org/apache/seatunnel/config/SeaTunnelConfigFactoryTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.config;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.SeaTunnelConfigFactory;
+import org.apache.seatunnel.config.utils.FileUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+public class SeaTunnelConfigFactoryTest {
+
+    @Test
+    public void testBasicParseAppConf() {
+
+        Config config = SeaTunnelConfigFactory.parseFile(FileUtils.getFileFromResources("factory/app.conf"));
+
+        Assert.assertTrue(config.hasPath("env"));
+        Assert.assertTrue(config.hasPath("source"));
+        Assert.assertTrue(config.hasPath("transform"));
+        Assert.assertTrue(config.hasPath("sink"));
+
+    }
+
+    @Test
+    public void testTransformOrder() {
+
+        Config config = SeaTunnelConfigFactory.parseFile(FileUtils.getFileFromResources("factory/app.conf"));
+
+        String[] pluginNames = {"split", "sql1", "sql2", "sql3", "json"};
+
+        List<? extends Config> transforms = config.getConfigList("transform");
+        for (int i = 0; i < transforms.size(); i++) {
```

Comment:
better to iterate pluginNames to prevent transforms's length is less than pluginNames.

Reply:
Thanks, add `Assert.assertEquals(pluginNames.length, transforms.size());`